op-devstack/presets/orchestrator.go

```diff
@@ -157,3 +165,26 @@ Add a TestMain to your test package init the orchestrator:
 	}
 	return out
 }
+
+// WithCompatibleTypes is a common option that can be used to ensure that the orchestrator is compatible with the preset.
+// If the orchestrator is not compatible, the test will either:
+// - fail with a non-zero exit code (42) if DEVNET_EXPECT_PRECONDITIONS_MET is non-empty
+// - skip the whole test otherwise
+// This is useful to ensure that the preset is only used with the correct orchestrator type.
+// Do yourself a favor, if you use this option, add a good comment (or a TODO) justifying it!
```

Comment:
Perhaps help set the precedent in this PR with your use of it in `op-devstack/presets/minimal_with_conductors.go`? 😄

Reply:
that's very fair :)
done